ui_tests/src/test/resources/com/saucedemo/account/SignInPositive.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Sign In
+  @signIn
+  Scenario: Sign In with valid username and password
```

Comment:
@Firdavs11 Please go through the scenario name and make them a bit more descriptive. 
It can be as simple as:

- Sign in as standart_user
- Sign in as problem_user
- etc.
but different